tests/benchmark/crud_test.go

```diff
@@ -0,0 +1,895 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package benchmark
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/rpc"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+func getRootResource() string {
+	registryProject, valueExists := os.LookupEnv("REGISTRY_PROJECT_IDENTIFIER")
+	if valueExists {
+		return fmt.Sprintf("projects/%s/locations/global", registryProject)
+	}
+	return "projects/bench/locations/global"
+}
+
+func getNumberOfVersions() int {
+	strNumberOfVersions, exists := os.LookupEnv("REGISTRY_VERSION_COUNT")
+	numberOfVersions, err := strconv.Atoi(strNumberOfVersions)
+	if err != nil || !exists {
+		numberOfVersions = 1
+	}
+	return numberOfVersions
+}
+
+func getApiName(i int) string {
+	apiNamePrefix, valueExists := os.LookupEnv("API_NAME_PREFIX")
+	if !valueExists {
+		apiNamePrefix = "benchtest"
+	}
+	apiNameStartOffset, valueExists := os.LookupEnv("API_NAME_START_OFFSET")
+	if valueExists {
+		intVar, _ := strconv.Atoi(apiNameStartOffset)
+		i += intVar
+	}
+
+	return fmt.Sprintf("%s-%d", apiNamePrefix, i)
+}
```

Comment:
Why do we need these to be customizable?

Reply:
Say I ran the create script and created 100 APIs. 
I wanted to add 100 more APIs to an existing environment without having to delete and rerun the script with 200 APIs

This was Needed when I started seeing errors while trying to adding 2000 APIs.
so I could break down the create into smaller chunks